bluesky_queueserver/server/server.py

```diff
@@ -110,20 +110,20 @@ class REResumeOptions(str, Enum):
 
 
 @app.get("/")
-async def _hello_handler():
+async def _ping_handler():
```

Comment:
Maybe we don't need these handler functions to be private, as they are going to be used by FastAPI (only).

Reply:
Removed the underscores.

Reply:
Thanks!